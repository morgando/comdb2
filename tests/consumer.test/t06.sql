drop table if exists t
create table t(i int)$$
exec procedure sys.cmd.send('fail_to_create_default_cons 1');
create default lua consumer t_cons on (table t for insert and update and delete)
exec procedure sys.cmd.send('fail_to_create_default_cons 0');
select count(*) from comdb2_procedures where name="t_cons"
create default lua consumer t_cons on (table t for insert and update and delete)
select count(*) from comdb2_procedures where name="t_cons"
