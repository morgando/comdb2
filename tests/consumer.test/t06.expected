(out='Tunable 'fail_to_create_default_cons' set to ON')
[create default lua consumer t_cons on (table t for insert and update and delete)] failed with rc 240 comdb2 default consumer 1.1
(out='Tunable 'fail_to_create_default_cons' set to OFF')
(count(*)=0)
(count(*)=1)
